src/Features/LanguageServer/ProtocolUnitTests/Definitions/GoToDefinitionTests.cs

```diff
@@ -37,6 +37,8 @@ void M()
             await using var testLspServer = await CreateTestLspServerAsync(markup, mutatingLspWorkspace);
 
             var results = await RunGotoDefinitionAsync(testLspServer, testLspServer.GetLocations("caret").Single());
+            // Verify that as originally serialized, the URI had a file scheme.
+            Assert.True(results.Single().Uri.OriginalString.StartsWith("file"));
```

Comment:
I don't think this is necessarily correct assertion. 
It is entirely correct to create `Uri` instance using local path: `new Uri("C:\a.cs")`. This gets internally translated to `file:///C:/a.as`, so the `AbsoluteUri` would be fine but `OriginalString` would be `C:\a.cs`.

Reply:
This is the correct assertion for this particular test.  The go to definition result is in a file - the json string it serializes should be a file schemed URI.

The test client will then convert the server's json response into LSP types.  So the original string it uses should be the file schemed URI that it got from the server.